collect_app/src/test/java/org/odk/collect/android/support/FormUtils.java

```diff
@@ -28,12 +28,12 @@ public static String createXForm(String id, String version) {
                 "</h:html>";
     }
 
-    public static Form.Builder buildForm(long id, String jrFormId, String jrVersion, String formFilesPath) {
-        return buildForm(id, jrFormId, jrVersion, formFilesPath, "blah");
+    public static Form.Builder buildForm(long id, String formId, String version, String formFilesPath) {
```

Comment:
I have no idea why these values ended up being named `jrFormId` and `jrVersion`. The irony is that JavaRosa actually has no concept of them, they are only meaningful at the Collect level. They're clearly an ODK form spec thing at least at this point and the `jr` prefix is confusing.